Stats block is visible on wider screens

stanislav-slavov opened the pull request:
ref #298
OP task: #603

venimus commented:
Btw https://github.com/aeternity/superhero-ui/pull/299/files#diff-64fb8a23b2e52577d4721f1f3eb493f6R156 (`.overview__label`) seems this was a wrong value initially. It should be `$tip_note_color`. Can you change it above (can't comment it) in the scope of this issue.

Else I think designers have to check that afterwards, because I suspect font sizes (spacing) should have some who-knows proportions (VALUE/label).

stanislav-slavov commented:
> Btw https://github.com/aeternity/superhero-ui/pull/299/files#diff-64fb8a23b2e52577d4721f1f3eb493f6R156 (`.overview__label`) seems this was a wrong value initially. It should be `$tip_note_color`. Can you change it above (can't comment it) in the scope of this issue.
> 
> ![image](https://user-images.githubusercontent.com/632282/78588983-2ca1c880-7848-11ea-956f-050b019d9aeb.png)
> 
> 1. Designed   2. Implemented
> 
> Else I think designers have to check that afterwards, because I suspect font sizes (spacing) should have some who-knows proportions (VALUE/label).

Added $tip_note_color
